src/common/CommandLine.cs

```diff
@@ -1746,18 +1746,18 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash;
+        private bool _qualifiersUseOnlyDash = true;
```

Comment:
This is the only interesting change in this diff. Everything else is just line-ending being removed by sublime text.